exporter/splunkhecexporter/config_test.go

```diff
@@ -90,6 +91,16 @@ func TestLoadConfig(t *testing.T) {
 					NumConsumers: 2,
 					QueueSize:    10,
 				},
+				BatcherConfig: exporterbatcher.Config{
```

Comment:
Is there a test that ensures the batching functionality is disabled by default?

Reply:
It is tested as part of base test case(defaultCfg). Do you think that is sufficient?